clients/netcore/Tributech.Dsk.Api.Clients.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
 
     <PackageId>Tributech.Dsk.Api.Clients</PackageId>
-    <Version>1.9.1-rc0</Version>
+    <Version>1.9.1-rc1</Version>
```

Comment:
The release branch name (and PR name) doesn't match the actual version.
If we delete the release branch afterwards  (and rename the PR) it would be ok for me to simply merge it like it is.

Reply:
I renamed the PR.
As discussed directly...please delete the release branch after you merged to master and develop since it might be misleading!